makefile

```diff
@@ -15,6 +15,23 @@ include make/default_compiler_options
 MATH ?=
 include make/libraries
 
+##
+# Set default compiler options.
+##
```

Comment:
This stuff is now included above from `make/default_compiler_options`; make sure you fix this to use those instead before merging/

Reply:
done